README.md

```diff
@@ -76,5 +75,6 @@ Usage: mmdc [options]
     -b, --backgroundColor [backgroundColor]  Background color. Example: transparent, red, '#F0F0F0'. Optional. Default: white
     -c, --configFile [config]                JSON configuration file for mermaid. Optional
     -C, --cssFile [cssFile]                  CSS alternate file for mermaid. Optional
+    -T, --customTheme <customThemeCssFile>   CSS file replacing CSS injected into SVG container. Optional
```

Comment:
Not saying `<...>` is wrong but maybe `[customThemeCssFile]` for consistency?

Reply:
Yea - I was weighing consistency over correctness here.  If you use the `<...>` version, then commander stops the run and reports the missing option.  My inclination was to change all the others to `<...>` but that seemed presumptuous to do as part of my change.

That said, I'm fine with switching to the `[...]` version if you like.  I'd just want to make sure that the application handles a missing file name gracefully.

Reply:
FYI - I just tested using one of the existing options.  As it stands, if you just specify one of the options which expects a file but don't provide one, you get this.

```
$  node index.js -C -i test/flowchart.mmd

CSS file "true" doesn't exist
```

Reply:
I'm with you regarding changing all, @tylerlong is the developer I just proposed a couple improvements like you did.

Reply:
OK - I've included that in this PR now as well.

Reply:
I don't get the point. If you specify `-C`, you should specify a CSS file such as `-C ./test/index.css`.

Reply:
I don't have any issue with the current commander code. You can omit any options except those marked as `required`.